src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String SEPARATE_SYMBOL = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String [] data = readFromFile(fromFileName);
+        writeToFile(data, toFileName);
+    }
+
+    private String [] readFromFile(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(SEPARATE_SYMBOL);
+                value = bufferedReader.readLine();
+            }
+            return stringBuilder.toString().split(SEPARATE_SYMBOL);
```

Comment:
Although this code is working, It's an unclear solution. You have an array of strings like {"buy", "2", "sell", "3"} and this array is very fragile. If someone will change order (use sort for example), your solution will stop working.

I suggest rewriting readFromFile method to return such array {"buy,2", "sell,3"}